tokio/src/sync/notify.rs

```diff
@@ -222,8 +222,11 @@ struct Waiter {
     /// Waiting task's waker.
     waker: Option<Waker>,
 
-    /// `true` if the notification has been assigned to this waiter.
-    notified: Option<NotificationType>,
```

Comment:
Renamed this field because it was confusing to me because of the similarly named future.